app/scripts/views/reset_password.js

```diff
@@ -54,9 +48,7 @@ define(function (require, exports, module) {
             this.model.set('error', err);
           });
       }
-
-      return super.beforeRender();
```

Comment:
To future proof ourselves, I think there should be a `return FormView.prototype.beforeRender.call(this);` here.